pkgs/by-name/ka/kanidm/package.nix

```diff
@@ -13,6 +13,10 @@
 , pam
 , bashInteractive
 , rust-jemalloc-sys
+, kanidm
+# If this is enabled, kanidm will be built with two patches allowing both
+# oauth2 basic secrets and admin credentials to be provisioned
+, enableSecretProvisioning ? false
```

Comment:
I’d like to see unsupported in the name of this option. I know that’s a long word to add but we should be clear that this is something upstream fundamentally opposes.

Reply:
Yes, this is definitely a good idea. Any ideas on how we can make sure that users know that any potential problems with provisioning should not be reported upstream?

Reply:
Is there any place where these options are visible except in the source here?
If it’s only here, I think a comment is fine. A link to the (closed) upstream issue would also be nice.

Reply:
I've added a comment for now, including upstream link and a notice for where to report issues with provisioning.